introduction/faq.md

```diff
@@ -367,46 +367,43 @@ This also means that it is possible to update the software for several qubes sim
 
 ### How much memory is recommended for Qubes?
 
-At least 6 GB, but 8 GB is more realistic. 
-It is possible to install Qubes on a system with 4 GB of RAM, but the system would probably not be able to run more than three qubes at a time.
+Please see the [system requirements].
 
-### Can I install Qubes 4.x on a system without VT-x or VT-d?
+### Can I install Qubes on a system without VT-x/AMD-V or VT-d/ADM-Vi/AMD IOMMU?
 
-Qubes 4.x requires Intel VT-x with EPT / AMD-V with RVI (SLAT) and Intel VT-d / AMD-Vi (aka AMD IOMMU) for proper functionality (see the [4.x System Requirements]).
+Please see the [system requirements] for the latest information.
 If you are receiving an error message on install saying your "hardware lacks the features required to proceed", check to make sure the virtualization options are enabled in your BIOS/UEFI configuration.
 You may be able to install without the required CPU features for testing purposes only, but VMs (in particular, sys-net) may not function correctly and there will be no security isolation. 
-For more information, see our post on [updated requirements for Qubes-certified hardware](/news/2016/07/21/new-hw-certification-for-q4/).
+For more information, see [Qubes-certified hardware](/doc/certified-hardware/).
 
-### Can I install Qubes OS on a system without VT-x?
+### Why is VT-x/AMD-V important?
 
-Yes, for releases 3.2.1 and below. 
-Xen doesn't use VT-x (or AMD-v) for PV guest virtualization. 
+Xen doesn't use VT-x/AMD-V for PV guest virtualization. 
 (It uses ring0/3 separation instead.) 
-However, without VT-x, you won't be able to use fully virtualized VMs (e.g., Windows-based qubes), which were introduced in Qubes 2. 
-In addition, if your system lacks VT-x, then it also lacks VT-d. (See next question.)
+However, without VT-x/AMD-V, you won't be able to use fully virtualized VMs (e.g., Windows-based qubes). 
```

Comment:
In 4.x we use HVM/PVH (which requires VT-x/AMD-V) for Linux VMs too, so this needs an update. Basically without VT-x/AMD-V no VM will start in default configuration.

Reply:
Thanks, updated.